recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/dm-salvator-x-m3n.cfg

```diff
@@ -0,0 +1,67 @@
+displays = (
+    {
+        name = "agl";
+        id = 0;
+    },
+    {
+        name = "android";
+        id = 1;
+    }
+);
+
+layers = (
+    {
+        name = "agl";
+        id = 1001;
+        display = "agl";
+        width = 768;
+        height = 700;
+        destination = {
+            x = 1;
+            y = 218;
+        }
+    },
+    {
+        name = "android";
+        id = 1000;
+        display = "android";
+        width = 1920;
+        height = 1080;
+    }
+);
+
+surfaces = (
+    {
+        name = "agl";
+        id = 1001;
+        layer = "agl";
+        width = 768;
+        height = 1024;
+        destination = {
+            width = 500;
+            height = 700;
+        }
+    },
+    {
+        name = "android";
+        id = 1000;
+        layer = "android";
+        width = 1920;
+        height = 1080;
+    }
+);
+
+events = (
+    {
+         event = "USER";
+         id = 1;
+         actions = (
+             {
+                 action = "ORDER";
+                 object = "LAYER";
+                 name = "agl";
+                 order = 0;
+             }
+         );
+    }
+);
```

Comment:
I think this one is not needed as we don't use DM. This is related to all machines we are going to support in Gen3 test

Reply:
+1

Reply:
Done